packages/apollo-server-fastify/src/ApolloServer.ts

```diff
@@ -26,12 +34,15 @@ const stringifyHealthCheck = fastJson({
   },
 });
 
-export class ApolloServer extends ApolloServerBase {
+export class ApolloServer<
+  ContextFunctionParams = FastifyContext,
+> extends ApolloServerBase<ContextFunctionParams> {
   async createGraphQLServerOptions(
-    request?: FastifyRequest,
-    reply?: FastifyReply,
+    request: FastifyRequest,
+    reply: FastifyReply,
```

Comment:
they're not actually optional: https://github.com/apollographql/apollo-server/blob/48d3ab205afc682d5304c7c1cae349e0e2a1ffd5/packages/apollo-server-fastify/src/ApolloServer.ts#L113